lib/medtronic/cli/blob_loader.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+var program = require('commander');
+var fs = require('fs');
+
+var driverManager = require('../../driverManager');
+var pwdSimulator = require('../medtronicSimulator.js');
+var builder = require('../../objectBuilder.js')();
+var medtronicDriver = require('../../drivers/medtronicDriver');
+var api = require('../../core/api.js');
+var config = require('../../../.config.js');
+var pkg = require('./../../../package.json');
+
+var intro = 'Medtronic CLI:';
+
+
+/*
+ * Load the given medtronic blob and then parse and send the data to the tp-platform
+ */
+function loadFile(filePath, tz, userid) {
+  fs.readFile(filePath, 'utf8', function(error, data) {
+    console.log(intro, 'Reading', filePath);
+    if (error) {
+      console.log(intro, 'Error reading Medtronic blob', error);
+      return;
+    }
+
+    var json = JSON.parse(data);
```

Comment:
Yes. At the moment the "blob" is just an array of of arrays where I'm storing each page of binary data separately, but I'll be adding other metadata (e.g. model number) to the file in future, which is why I'm storing it as JSON.